### ISSUE TITLE
PR #240 added dependency on `addressable` but it isn't in the gemspec.

### ISSUE THREAD

**Issue by rally25rs** [auth0/ruby-auth0#246]
### Description

Version `4.17.0` of this gem now has a dependency on `addressable` due to #240 but no one added the dependency to the gemspec, so using it results in

```
LoadError: cannot load such file -- addressable/uri
/usr/local/bundle/gems/bootsnap-1.4.8/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:34:in `require'
...
/usr/local/bundle/gems/auth0-4.17.0/lib/auth0/mixins/httpproxy.rb:1:in `<main>'
...
/usr/local/bundle/gems/auth0-4.17.0/lib/auth0/mixins.rb:8:in `<main>'
...
/usr/local/bundle/gems/auth0-4.17.0/lib/auth0.rb:2:in `<main>'
...
```

Seems like `Gemfile` and `auth0.gemspec` should include `addressable` as dependencies.

### Reproduction

Use v4.17.0 of this gem in a Rails project that doesn't already have a dependency on `addressable`.

### Environment

> Please provide the following:

- **Version of this library used:** 4.17.0
- **Version of the platform or framework used, if applicable:**
- **Other relevant versions (language, server software, OS, browser):**
- **Other modules/plugins/libraries that might be involved:**

**Comment by davidpatrick** [comment]
Thanks for raising this @rally25rs, looks like it didn't make it into the Gemfile.  I will get this patched up right away

**Comment by davidpatrick** [comment]
Fixed in https://github.com/auth0/ruby-auth0/pull/247

4.17.1 on its way

**Comment by rally25rs** [comment]
Confirmed fixed in `4.17.1` 👍
